Assets/Settings/InputActions.inputactions

```diff
@@ -548,6 +548,11 @@
                     "devicePath": "<Keyboard>",
                     "isOptional": false,
                     "isOR": false
+                },
+                {
+                    "devicePath": "<Mouse>/{Back}",
```

Comment:
That one was weird, I though I could select the Mouse itself but I had to select an action from the Mouse